### PR TITLE
Adds pretrained IDs directly in the tests

### PR DESCRIPTION
**Pull request by LysandreJik**
Stop relying on pretrained maps to get the rust tokenizer. First PR to enable #29112

### CODE REVIEW

File: tests/test_tokenization_common.py [hunk 1]
```diff
@@ -200,19 +201,13 @@ def setUp(self) -> None:
...
+                    self.from_pretrained_id,
                     self.from_pretrained_kwargs if self.from_pretrained_kwargs is not None else {},
                 )
-                for pretrained_name in self.rust_tokenizer_class.pretrained_vocab_files_map[
-                    self.from_pretrained_vocab_key
-                ].keys()
-                if self.from_pretrained_filter is None
-                or (self.from_pretrained_filter is not None and self.from_pretrained_filter(pretrained_name))
             ]
-            self.tokenizers_list = tokenizers_list[:1]  # Let's just test the first pretrained vocab for speed
```
Comment (LysandreJik) [inline tests/test_tokenization_common.py:215]: @amyeroberts the gist of the changes is what is happening here. So far we're relying on this `self.rust_tokenizer_class.pretrained_vocab_files_map`, which returns the list of defined checkpoints from the vocab files maps.

We then only select the first one: `self.tokenizers_list = tokenizers_list[:1] `

With this change, we need to define, within testing, the repo ID that we'll use for integration tests. IMO it's much cleaner this way as we're not putting a list of random repo IDs within the user's runtime. This frees the way for the #29112 refactor to ship.

File: tests/models/bert_generation/test_tokenization_bert_generation.py [hunk 2]
```diff
@@ -29,6 +29,7 @@
 
 @require_sentencepiece
 class BertGenerationTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
+    from_pretrained_id = "bert_for_seq_generation"
```
Comment (amyeroberts) [inline tests/models/bert_generation/test_tokenization_bert_generation.py:32]: Do we not have a version of this checkpoint with an org prefix?

File: tests/models/clvp/test_tokenization_clvp.py [hunk 3]
```diff
@@ -25,6 +25,7 @@
 
 
 class ClvpTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
+    from_pretrained_id = "clvp_dev"
```
Comment (amyeroberts) [inline tests/models/clvp/test_tokenization_clvp.py:28]: Same q here

File: tests/models/ernie_m/test_tokenization_ernie_m.py [hunk 4]
```diff
@@ -28,6 +28,7 @@
 @require_sentencepiece
 @require_tokenizers
 class ErnieMTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
+    from_pretrained_id = "ernie-m-base"
```
Comment (amyeroberts) [inline tests/models/ernie_m/test_tokenization_ernie_m.py:31]: And here

File: tests/models/mgp_str/test_tokenization_mgp_str.py [hunk 5]
```diff
@@ -27,6 +27,7 @@
 
 @require_tokenizers
 class MgpstrTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
+    from_pretrained_id = "mgp-str"
```
Comment (amyeroberts) [inline tests/models/mgp_str/test_tokenization_mgp_str.py:30]: And here

File: tests/test_tokenization_common.py (hunk 1 above)
Comment (amyeroberts) [reply tests/test_tokenization_common.py:215]: Agreed - much cleaner!

**Review by amyeroberts — APPROVED** [review]
Great - thanks for adding this. 

Just a few small comments on checkpoints without orgs

**Review by ArthurZucker — COMMENTED** [review]
Great work!

**Pull request merged**